app/views/companies/_company_h_brand.html.haml

```diff
@@ -22,6 +21,9 @@
       %hr.company-info-separator
 
       %p.company-info
+
+        - company = Company.find(params[:id])
+
```

Comment:
We should not use the `params` hash to all in views - controllers, yes, but not views.

Also, `company` is already set as a local variable for this view in the `image_html` method in the `ImagesUtility` module.